add 0.11.0 release notes (#333)
weaveworks/pctl: docs/release_notes/v0.11.0.md

# Release 0.11.0
## Bug Fixes

- Specify the default values configmap in Kustomize file when creating helmchart artifacts with default values (#332)

> - Update command line parsing error (#329)

Author: @Skarlso copied from the auto-generated release notes, got a better message to put? This is a bit generic :smile:

Reviewer: How about this:

- Return an understandable error in case of missing command line arguments

> - Improve error message when arguments are missing during pctl add (#329)

Reviewer: ```suggestion
- Improve error message in case of missing arguments during pctl add (#329)
```
